Complete full, mobile and interim responsive views

smithWEBtek opened the pull request:
* Add css to stories.scss
* Add classes and content to stories.html.erb
* Add 'group-1111.png' background image to assets

Resolves #274 .

# Why is this change necessary?
This code provides the required view for the weigh-in landing page header.

# How does it address the issue?
The css styling, background image and css classes provide the required design in full desktop width, interim widths and mobile width.

# What side effects does it have?
None

mzagaja commented on vendor/extensions/stories/app/views/refinery/stories/stories/index.html.erb:
```diff
@@ -1,30 +1,42 @@
-    <div class="container">
-      <div class="content-column banner-content">
-        <h1>Weigh In</h1>
+    <div class="container"></div>
```
Right now this container class is unused and technically could be removed, but I would suggest maybe following the pattern we use on the home page. See https://github.com/MAPC/digitalhub/blob/develop/app/views/refinery/pages/home.html.erb#L14-L39 for that example.

mzagaja commented on vendor/extensions/stories/app/views/refinery/stories/stories/index.html.erb:
```diff
@@ -1,30 +1,42 @@
+          Opinions, ideas, and priorities that spring from the lived experience of the people of the region are an 
+          important part of MetroCommon. We’re collecting public input at every stage of the planning process.
+        <br />
+        <br />
```
Instead of using `<br />` here would you consider surrounding each paragraph in a `<p>` tag?

mzagaja commented on app/assets/stylesheets/refinery/stories.scss:
```diff
@@ -1,5 +1,44 @@
+    background-image: url('group-2111.png');
+    padding: 14.18rem 19.5rem 14.18rem 37.43rem;
+    
+    @media (max-width: 1600px){
```
Instead of using defined pixel values here maybe when you refactor to use container in the DOM, you can implement using the breakpoints we use in `_vars.scss` https://github.com/MAPC/digitalhub/blob/develop/app/assets/stylesheets/global/_vars.scss#L63-L67. See https://github.com/MAPC/digitalhub/blob/develop/app/assets/stylesheets/refinery/home.scss#L61-L109 for an example of how this is implemented. If standardizing these break points turns into a bear I'm happy to pair on this with you tomorrow morning.

mzagaja commented on app/assets/stylesheets/refinery/stories.scss:
```diff
@@ -1,5 +1,44 @@
+    
+    &--title {
+      color: #111436;
+      font-size: 2.125rem;
```
I've generally been defining font sizes and colors over at https://github.com/MAPC/digitalhub/blob/develop/app/assets/stylesheets/global/_vars.scss#L8-L16 and integrating these into the CSS file. I think you can potentially create vars and add these if they are new, but it might also be worth a discussion with Kit that each of these font sizes is different and see if it makes sense to standardize them a bit more.

smithWEBtek commented:
@mzagaja I have incorporated changes, retaining the existing pattern from the home page, incorporating existing vars where possible. I have added a color to _vars.scss since it did not exist and the XD file used it for the Weigh In title and text:  `$v3-color_black-dark: #111436;`.  I settled on using css grid for the layout, and incorporated existing media breaks from `_vars.scss`
